### PR TITLE
Map SUL/SDR location to SUL/INTERNET for parity

### LINKED ISSUE #938: Map SUL/SDR location to SUL/INTERNET for parity
Some electronic holdings in FOLIO have the location code SUL-SDR, but in Symphony were in SUL/INTERNET.

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #938

### CODE REVIEW

File: lib/folio/eresource_holdings_builder.rb [hunk 1]
```diff
@@ -85,7 +85,8 @@ def barcode(index)
     end
 
     def home_location_code
```
Comment (cbeer) [inline lib/folio/eresource_holdings_builder.rb:87]: Should we push this into `LocationsMap`, where we're already doing some of this location code massaging?

Comment (jcoyne) [reply lib/folio/eresource_holdings_builder.rb]: How about into locations.tsv?

Comment (cbeer) [reply lib/folio/eresource_holdings_builder.rb]: I thought we were trying to keep our locations.tsv in sync with what libsys is using.

File: lib/translation_maps/locations.tsv [hunk 2]
```diff
@@ -449,4 +449,4 @@ SSRC-FIC-S	GREEN	GRE-DOCS-MICROFICHE
 SDR	LAW	LAW-SDR
 SDR	BUSINESS	BUS-SDR
 SDR	HOOVER	HILA-SDR
-SDR	SUL	SUL-SDR
+INTERNET	SUL	SUL-SDR
```
Comment (cbeer) [inline lib/translation_maps/locations.tsv:452]: (Copied from the outdated thread)

I thought we were trying to keep our locations.tsv in sync with what libsys is using.

Comment (jcoyne) [reply lib/translation_maps/locations.tsv:452]: 🤷 are we documenting that somewhere?

Comment (cbeer) [reply lib/translation_maps/locations.tsv:452]: Sure, anywhere you think would be appropriate. It seems like a very good idea not to change that upstream data.

**Review by cbeer — APPROVED** [review]

**Pull request merged**